include/build_config/SDL_build_config.h.cmake

```diff
@@ -208,6 +208,12 @@
 #cmakedefine HAVE_POLL 1
 #cmakedefine HAVE__EXIT 1
 
+#if !defined(__linux__)
+/* Disabling for Linux at least for now until the feature matures
+   a bit on this platform. */
```

Comment:
This doesn't look good. I know @madebr expressed concerns about compatibility
of library built on newer systems would be inoperable on older ones, but that
is a well known story anyway.  So, if this patch is going to go in, and if we
really want to address that, I suggest making this a compile time option in
cmake rather than explicitly disabling it for linux.

Besides, this part of the code isn't even enabled by default, so I don't know
why is there sof much effont on it...

Reply:
Yes, in general we do build-time checks for system functionality and we don't guarantee that something built on one version of Linux will work on an earlier version.

To @sezero's point, is this code that will be run in SDL?

Reply:
> To @sezero's point, is this code that will be run in SDL?

Unless SDL_malloc.c is explicitly patched, no: The patch changes code guarded
by `FOOTERS` which isn't defined by default (neither is `USE_DEV_RANDOM`)